resources/build/carthage-workaround.sh

```diff
@@ -0,0 +1,23 @@
+# carthage.sh
+# Usage example: ./carthage.sh build --platform iOS
```

Comment:
nit - aren't these carthage-workaround.sh?

Reply:
Aside from the comment lines after these, it's a direct copy-paste from the code specified here:  https://github.com/Carthage/Carthage/blob/master/Documentation/Xcode12Workaround.md

This is the workaround we have mentioned in the iOS readme file.